Lib/fontTools/cffLib.py

```diff
@@ -1293,13 +1726,25 @@ def xmlRead(self, name, attrs, content, parent):
 	((12, 32),	'CIDFontRevision',	'number',	0,	None),
 	((12, 33),	'CIDFontType',		'number',	0,	None),
 	((12, 34),	'CIDCount',		'number',	8720,	None),
-	(15,		'charset',		'number',	0,	CharsetConverter()),
+	(15,		'charset',		'number',	None,	CharsetConverter()),
 	((12, 35),	'UIDBase',		'number',	None,	None),
 	(16,		'Encoding',		'number',	0,	EncodingConverter()),
 	(18,		'Private',	('number', 'number'),	None,	PrivateDictConverter()),
 	((12, 37),	'FDSelect',		'number',	None,	FDSelectConverter()),
 	((12, 36),	'FDArray',		'number',	None,	FDArrayConverter()),
 	(17,		'CharStrings',		'number',	None,	CharStringsConverter()),
+	(24,		'VarStore',		'number',	None,	VarStoreConverter()),
+	(25,		'maxstack',		'number',	None,	None),
+]
+
+topDictOperators2 = [
+#	opcode		name			argument type	default	converter
+	((12, 7),	'FontMatrix',		'array',	[0.001, 0, 0, 0.001, 0, 0],	None),
+	((12, 37),	'FDSelect',		'number',	None,	FDSelectConverter()),
+	((12, 36),	'FDArray',		'number',	None,	FDArrayConverter()),
+	(17,		'CharStrings',		'number',	None,	CharStringsConverter()),
+	(24,		'VarStore',		'number',	None,	VarStoreConverter()),
+	(25,		'maxstack',		'number',	None,	None),
```

Comment:
According to [the spec](https://www.microsoft.com/typography/otspec/cff2.htm#section7), `vstore` instead of `VarStore`, and default value of `maxstack` is 193.